warehouse/pom.xml

```diff
@@ -475,6 +475,18 @@
                             <log4j.configuration>file:${project.build.testOutputDirectory}/log4j.properties</log4j.configuration>
                         </systemPropertyVariables>
                     </configuration>
+                    <dependencies>
+                        <dependency>
+                            <groupId>org.junit.jupiter</groupId>
+                            <artifactId>junit-jupiter-engine</artifactId>
+                            <version>${version.junit.bom}</version>
```

Comment:
isn't this already defined in the parent pom?

Reply:
Plugin dependency management isn't a thing, unfortunately.  If you want to avoid having to redefine this plugin in multiple poms you could just move the maven-surefire-plugin configuration from the warehouse and web-services poms to the pluginManagement entry for the datawave parent pom.  That way, all of the child modules will inherit the same surefire configuration, and you will only have to define it in one place.

Honestly, I'm not sure why we aren't already doing that.  The surefire plugin configuration for warehouse and web-services is identical.